samples/gen-go-blog/server/router.go

```diff
@@ -46,10 +46,13 @@ type Server struct {
 
 // Serve starts the server. It will return if an error occurs.
 func (s *Server) Serve() error {
+	tracingToken := os.Getenv("TRACING_ACCESS_TOKEN")
+	ingestURL := os.Getenv("TRACING_INGEST_URL")
+	isLocal := os.Getenv("_IS_LOCAL") == "true"
 
-	go func() {
-		metrics.Log("blog", 1*time.Minute)
-	}()
+	if !isLocal {
+		startLoggingProcessMetrics()
+	}
```

Comment:
nit: why the extra function call instead of calling go func()ing `metrics.Log()`?

not a nit but actual correctness issue: I'm pretty sure the call to `metrics.Log` still needs to happen in a goroutine since it blocks: https://github.com/Clever/go-process-metrics/blob/95df732d99126d01eaf230349f3b3a5d9fcab7fc/metrics/metrics.go#L19-L69

Reply:
Doh, I didn't mean to remove the `go` call 🤦‍♂